### DISCUSSION TITLE
Cloud deployment?

### DISCUSSION THREAD

**Discussion by nsnydr** [pyannote/pyannote-audio#1534] [Q&A] [answered]
Hi, 

I know that pyannote does not process streams. But, despite knowing that, I am going to ask... is there a way to deploy it with a cloud provider environment (AWS/Azure/GCP) and use a cloud storage and compute for processing via a web app? I think the answer is no, but want to confirm as I know I have not figured it out in the documentation or through my own tests the past few days.

**Comment by hbredin** [comment, marked as answer]
Not sure what you are looking for exactly. Can you clarify?

`pyannote` is already used in production by a bunch of companies (and most likely deployed on cloud instances). 

If you are looking for a one-click pyannote-as-a-service deployment solution, I am not aware of any... except maybe something like [that](https://replicate.com/meronym/speaker-diarization) ?
